src/api/order.ts

```diff
@@ -0,0 +1,28 @@
+import type { OrderRequest, OrderResponse, OrderError } from "../types/product";
+
+export const createOrder = async (
+  orderData: OrderRequest,
+  authToken: string,
+): Promise<OrderResponse> => {
+  const response = await fetch("/api/order", {
+    method: "POST",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${authToken}`,

```

Comment:
주문이 실패해서 이유를 보니 토큰을 잘못보내고 있었네요.! 아래와 같이 수정해주시면 성공적으로 보내지게 됩니다 😊

```suggestion
      Authorization: authToken,
```